module/cmd/gravity/cmd/eth_keys.go

```diff
@@ -110,7 +113,10 @@ func runAddCmd(cmd *cobra.Command, args []string) error {
 }
 
 func printCreate(cmd *cobra.Command, keyOutput EthereumKeyOutput) error {
-	output, _ := cmd.Flags().GetString(cli.OutputFlag)
+	output, errFlagString := cmd.Flags().GetString(cli.OutputFlag)
+	if errFlagString != nil {
+		panic("ErrFlagString issue")
```

Comment:
Can we put the actual string in the panic message?

Reply:
Also can you change the name of the err variable, I think errFlagString isn't a very helpful variable name. Above you used errDryRun and I think something more similar to that regarding this particular flag would be nice.